### PR TITLE
Fix autotuner warnings

### PR DESCRIPTION
**Pull request by alihassanijr**
Some warnings are raised even when they're not supposed to.

Addresses #137.

CC @stevenwalton

### CODE REVIEW

**Comment by stevenwalton** [comment]
Looks good. I don't see any others that would present an issue. I'll let you know if I find more while testing. 

The thing I would change is providing a link to the documentation. I'd add that to the sections you modified and line 140

**Comment by stevenwalton** [comment]
I think https://github.com/SHI-Labs/NATTEN/blob/main/docs/fna/autotuner.md is better than https://shi-labs.com/natten

Someone is going to come here and be confused.

**Comment by alihassanijr** [comment]
I think you might be looking at the wrong change; that was just a rewording of the error message for when it fails to dlopen the .so

I'm pointing to https://github.com/SHI-Labs/NATTEN/blob/main/docs/fna/autotuner.md in autotuner related warnings.

File: src/natten/functional.py
Comment (stevenwalton) [inline src/natten/functional.py:1]: https://github.com/SHI-Labs/NATTEN/blob/main/docs/build.md ?

But I think this is fine

File: src/natten/nested.py
Comment (stevenwalton) [inline src/natten/nested.py:1]: Ditto

**Review by stevenwalton — APPROVED** [review]
I think this is fine. You were right that I just looked at the first few

**Pull request merged**